config/config.json.in

```diff
@@ -29,6 +29,7 @@
     }
   },
   "memory": {
+    "pa_bits": @CONFIG__PA__BITS@,
```

Comment:
Also, I feel like `pa_bits` isn't immediately obvious and we don't need to be terse here. Can this be `physaddr_bits`? Then it nicely matches the type name. In fact... should we use this to drive the size of the type? Maybe that can be a future PR.